src/wovnio/html/HtmlConverter.php

```diff
@@ -370,6 +374,29 @@ private function putReplaceMarker($element, $marker)
         $element->innertext = $key;
     }
 
+    /**
+     * Note: Because php5.3 doesn't allow calling private method inside anonymous function,
+     * Use `_` prefix to imply `private`
+     */
+    public function _translateMetaTagLink($meta_node)
+    {
+        $httpEquiv = $meta_node->getAttribute('http-equiv');
+        $metaContent = $meta_node->getAttribute('content');
+
+        if ($httpEquiv === 'refresh') {
+            $splitMetaContent = preg_split('/;(\s*url)=/i', $metaContent, null, PREG_SPLIT_NO_EMPTY | PREG_SPLIT_DELIM_CAPTURE);
```

Comment:
Should we ignore non-numeric refresh time?
```suggestion
            $splitMetaContent = preg_split('/(\d+);(\s*url)=/i', $metaContent, null, PREG_SPLIT_NO_EMPTY | PREG_SPLIT_DELIM_CAPTURE);
```

Reply:
for our purposes we don't care, we are only interested in the url part regardless of how the browser processes the directive later